package.json

```diff
@@ -68,7 +68,7 @@
     "fhir": "^4.8.3",
     "flat": "^5.0.2",
     "fs-extra": "^9.0.1",
-    "fsh-sushi": "^2.0.0-beta.1",
+    "fsh-sushi": "^2.0.0-beta.3",
```

Comment:
I think the package-lock.json is still a little haywire. If I check out this branch and `npm install`, I get a `package-lock.json` with changes. When you run `npm install`, does your lock file change as well? If so I think we want to go ahead and commit those changes.

Reply:
I don't get any changes. I think the issue is that I'm running npm version 7 which creates a `package-lock.json` in a format that's ever so slightly different (which can be viewed as the `lockfileVersion` property at the top of the file). This new version is apparently backwards compatible, but I can downgrade my npm version and run `npm install` again for consistency's sake.